electron/games.ts

```diff
@@ -101,16 +136,31 @@ class LegendaryGame {
     }
   }
 
+  /**
+   * Alias for `GameConfig.get(this.appName).config`
+   *
+   * @returns GameConfig
+   */
   public async getSettings() {
-    return await GameConfig.get(this.appName).config
+    return GameConfig.get(this.appName).config
   }
 
+  /**
+   * Helper for `checkGameUpdates().contains(this.appName)`
+   *
+   * @returns If game has an update.
+   */
   public async hasUpdate() {
-    return (await Library.get().listUpdateableGames()).find((app_name) => {
-      return app_name == this.appName
-    }) !== undefined
+    return (await Library.get().listUpdateableGames()).indexOf(this.appName) >= 0
```

Comment:
indexOf could be replaced with a includes here. More readable and uses ES6.
so `.includes(this.appName)`

Reply:
I forgot that function existed. Jeez I need coffee.